.github/workflows/publish_webcomponents.yml

```diff
@@ -0,0 +1,68 @@
+name: Publish to Dotcms Webcomponents
+on:
+    push:
+        branches:
+            - release-*
+            - master
+jobs:
+    publish:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout core-web
+              uses: actions/checkout@v2
+              with:
+                  fetch-depth: 1
+                  path: 'core-web'
+            - name: Configuring Node.js
+              uses: actions/setup-node@v2-beta
+              with:
+                  node-version: '10.15.3'
+            - name: Installing dependencies
+              run: |
+                  cd core-web
+                  npm install
+            - name: 'Automated Version Bump - dotcms-webcomponents'
+              id: version-bump
+              uses: 'phips28/gh-action-bump-version@master'
+              with:
+                  commit-message: 'CI: bumps version to {{version}} [skip ci]'
+                  skip-tag: 'true'
+              env:
+                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+                  PACKAGEJSON_DIR: 'core-web/libs/dotcms-webcomponents'
+            - name: Build dotcms-webcomponents with deps
+              run: |
+                  cd core-web
+                  npm run nx build dotcms-ui -- --with-deps --prod
+            - name: 'publish to npm'
+              run: |
+                  cd core-web
+                  npm set //registry.npmjs.org/:_authToken ${{ secrets.NPM_TOKEN }}
+                  cd dist/libs/dotcms-webcomponents
+                  npm publish
+            - name: Checkout Core & Bump webComponentsReleaseVersion, only on a release branch
+              env:
+                  # TODO: Change me to someone else's username
+                  GITHUB_USER: victoralfaro-dotcms
```

Comment:
is it ok to have this username? shouldn't we use a company's generic username? How is it managed for others actions?

Reply:
this will be changed later